### DISCUSSION TITLE
Should I be able to delete empty Asset folders?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [craftcms/cms#12958] [Q&A] [answered]
I'm not sure if I'm doing something wrong but I can't delete empty folders, see this video https://media.yep.digital/ZJMXNhkv

**Comment by brandonkelly** [comment, marked as answer]
That action is only for assets. To delete a folder, navigate into it, and then delete it via its source path action.

<img width="231" alt="A subfolder’s source path actions, including a “Delete folder” option." src="https://user-images.githubusercontent.com/47792/228428423-2cd41260-18c3-4c28-b394-a17e7197ac43.png">

**Reply by RyanRoberts** [reply]
Hot damn!

**Comment by thomasbendl** [comment]
For me also it was not intuitively clear - how to delete a subfolder. It should also be possible to delete subfolders in action menu (same as for assets) - see screenshot.

![Bildschirmfoto 2023-03-30 um 09 05 28](https://user-images.githubusercontent.com/8804216/228758081-c46b1916-6750-4de2-b6f9-f817b037a7b4.png)

One other problem could be that when you create the first subfolder at root you only have one menu entry "New subfolder". So you think there are no further possibilities for folders. Maybe there should be already all possibilities. Which do not work grayed out.
